assets/app/scripts/directives/logViewer.js

```diff
@@ -216,7 +216,7 @@ angular.module('openshiftConsole')
               streamer.start();
             };
 
-            $scope.$watchGroup(['name', 'options.container'], streamLogs);
+            $scope.$watchGroup(['name', 'options.container', 'run'], streamLogs);
```

Comment:
Will restream the entire log in some cases? For instance, when a pod goes from running -> succeeded?